server/lib/service/dv360.dart

```diff
@@ -31,11 +34,58 @@ class DisplayVideo360Client {
         updateMask: 'entityStatus');
   }
 
+  /// Duplicates the line item to [insertionOrderIdDestination].
+  ///
+  /// The [advertiserIdDestination] is the ID of the destination advertiser.
+  ///
+  /// Throws an [ApiRequestError] if API returns an error.
+  Future<void> duplicateLineItem(Int64 advertiserId, Int64 lineItemId,
```

Comment:
fyi: a complete line item contains targeting which is a major part. but this requires much extra effort. so we are good now.